components/molecules/GisBenefitCard.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const GisBenefitCard = (props) => {
+  return (
+    <div className="grid grid-cols-1 divide-y divide-grey-500">
+      <section></section>
+      <section className="mt-4 mx-8" id="paymentDate">
+        <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-2 sm:gap-4 sm:mx-auto sm:mt-4">
+          <div id="applicationDate">
+            <p>Application Date</p>
```

Comment:
Do you mind adding locale support for all these texts?

Reply:
Done!